.prow/push-core-go-test.yaml

```diff
@@ -8,7 +8,19 @@ presubmits:
           command: [ "bash", "-c" ]
           args:
             - >
-              go mod download &&
-              make gen &&
-              git diff --exit-code &&
-              go test ./pkg/...
+              echo "Creating user" &&
```

Comment:
i installed `embedded-postgresql` go package for tests. Though, it's impossible to run as root, hence the modification here.